packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -864,6 +870,465 @@ describe('Table', () => {
                 verifyRenderedData(sortedData);
             });
         });
+
+        describe('hierarchical data', () => {
```

Comment:
I scanned the table docs for features that might be impacted by hierarchy. The only one I could think of that I didn't see tested was column order. If I change which column is first, it should affect which column shows the expand/collapse UI. I verified that this works in your Storybook build; should we have a unit test for it? Hopefully there are existing column re-order tests you could leverage from.

Reply:
So, column order does _not_ affect the expand/collapse UI as it is now being completely handled by the `Row` component (despite my original implementation).